public/js/src/module/gui.js

```diff
@@ -486,6 +488,9 @@ function printOcForm() {
                         }
                         if ( format.queries === 'yes' ) {
                             printified = $dn.trigger( 'printify.enketo' );
```

Comment:
Please also use the new event for the DN questions. This means changing the listener in dn-widget.js as well.

Reply:
code updated